conf.d/health.d/beanstalkd.conf

```diff
@@ -0,0 +1,36 @@
+# get the number of buried jobs in all queues
+
+template: server_buried_jobs
+      on: beanstalk.current_jobs
+    calc: $buried
+   units: jobs
+   every: 10s
+    warn: $this > 0
+    crit: $this > 10
+   delay: up 30s down 30s multiplier x2 max 5m
```

Comment:
hm... I think this should be:

```
delay: up 0 down 5m multiplier 1.2 max 1h
```

In detail:

- `up 0` get notified immediately when this alarm raises
- `down 5m` get clear notifications 5 minutes after they are clear
- `multiplier 1.2` if it flips again while we have delayed a notification, multiply the numbers above by 1.2
- `max 1h` even if it flips flops continuously, send one notification every hour